README.md

```diff
@@ -72,7 +72,7 @@ To run unit tests:
 cargo test
 ```
 
-Note that the `astria-proto` generates its code by running tests (and verifying
+Note that the `astria-core` generates its code by running tests (and verifying
```

Comment:
This part of the README is outdated and can be removed. `astria-core` no longer does this.